CHANGELOG.md

```diff
@@ -94,6 +94,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Removed
 
 ### Fixed
+- Added depth check in doc parser for deep nested document ([#5199](https://github.com/opensearch-project/OpenSearch/pull/5199))
```

Comment:
Need to remove the duplicate entry from line 71

Reply:
Thanks. Updated.